test/alternator/suite.yaml

```diff
@@ -1 +1,24 @@
-type: Run
+type: Python
+pool_size: 4
+prepare_cql: INSERT INTO system_auth_v2.roles (role, salted_hash) VALUES ('alternator', 'secret_pass')
+run_first:
+    - test_streams
+    - test_scan
+    - test_tracing
+    - test_ttl
+extra_scylla_config_options:
+  {
+    experimental_features: [
+                             udf,
```

Comment:
udf is a CQL-specific feature isn't needed for Alternator tests. So is keyspace-storage-options. Probably also "broadcast tables", whatever those are, are not related.

Reply:
1. You are executing the alternator test with these two options enabled
https://github.com/scylladb/scylladb/blob/03818c4aa96de33b40a336278dfb99bd797f6b53/test/cql-pytest/run.py#L313
https://github.com/scylladb/scylladb/blob/03818c4aa96de33b40a336278dfb99bd797f6b53/test/cql-pytest/run.py#L314
2. You can not enable enable_user_defined_functions, which you are using in alternator tests as well, without enabling udf experimental feature
https://github.com/scylladb/scylladb/blob/03818c4aa96de33b40a336278dfb99bd797f6b53/test/cql-pytest/run.py#L316
https://github.com/scylladb/scylladb/blob/03818c4aa96de33b40a336278dfb99bd797f6b53/gms/feature_service.cc#L70
4.broadcast-tables is deleted

Reply:
When test/cql-pytest/run.py's run_scylla was used for all Scylla-running - both CQL and Alternator, I didn't mind listing options which weren't needed for Alternator - they don't hurt. test.py also has a bunch of options they turn on for all Scylla runs. But now that you have an Alternator-specific file and are trying to list all the options just for the Alternator run - it's silly to list options that we don't need.

"user defined functions" is a CQL feature, that has nothing to do with Alternator. Are you sure we need them in the Alternator test suite?

Reply:
I'm migrating the alternator test from one type to another. So I'm just taking the parameters that you are using in run, as simple as that. As we will have a separate suite file, and we are using an iterative approach for changes, fine-tuning the parameters can be done in the next PR. Considering that the test's author has better knowledge of what parameters are required and what can be thrown away, he can do it in a few minutes. But we should also consider that there is a default configuration for test.py's PythonTestSuite, and we are adding/changing values in our file.

Reply:
I see, ok, that's indeed one way to go. My thinking was that you're adding **new** code - even if it was based on old code - and in that case, the new code needs to make sense and shouldn't blindly copy silly things from the old code, but I guess a valid approach is to copy - and only later fix the weird stuff. The only problem is that we often forget to do that "later" stuff.
So I'm resolving this conversation.